frontend/pages/home_page.js

```diff
@@ -27,20 +35,47 @@ export default class HomePage extends Component {
     this.fetchProducts();
   }
 
+  productStatusIncludesFilter = (string) => {
```

Comment:
i like to use 'named params' in all methods because it allows me to change the params in the function declaration without having to change any method implementations:

```
function func ({ param1 }) { ... }
func({ param1: 'value' });
```

later i change func:
```
function func ({ param2, param1 }) { ... }
// still works
func({ param1: 'value' });
```

because basically we're passing an object as a param, so it doesnt matter how many params there are and what order the params are in